device-types/Cisco/ISR2951.yaml

```diff
@@ -1,10 +1,10 @@
 ---
 manufacturer: Cisco
-model: 2951 ISR
-part_number: CISCO2951/K9
-slug: cisco-2951-isr
+model: ISR2951
+part_number: ISR2951/K9
```

Comment:
Been awhile but I am pretty sure this actually comes up as "CISCO2951", similar to my 2901's that come up as "CISCO2901/K9"